machine_management/creating-infrastructure-machinesets.adoc

```diff
@@ -36,7 +36,9 @@ include::modules/machineset-yaml-aws.adoc[leveloffset=+3]
 Machine sets running on AWS support non-guaranteed xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-non-guaranteed-instance_creating-machineset-aws[Spot Instances]. You can save on costs by using Spot Instances at a lower price compared to
 On-Demand Instances on AWS. xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-creating-non-guaranteed-instance_creating-machineset-aws[Configure Spot Instances] by adding `spotMarketOptions` to the `MachineSet` YAML file.
 
+:context: creating-machineset-azure
 include::modules/machineset-yaml-azure.adoc[leveloffset=+3]
+:context!: creating-machineset-azure
```

Comment:
You unset context here, but you never reset the original context that you start at the top of the assembly.